File: C#/seleniumTestName.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;
using System.Threading;
class EntryPoint
{
    static void Main()
    {
        IWebDriver driver = new ChromeDriver();

        driver.Navigate().GoToUrl("https://testing.todorvachev.com/selectors/name");

        IWebElement element = driver.FindElement(By.Name("myName"));

        if (element.Displayed)
        {
            Console.WriteLine("Very good! The element appears on the webpage.");
        } else
        {
            Console.WriteLine("Not good... The element doesn't appear on the webpage");
        }

        Thread.Sleep(6000);

        driver.Quit();
    }
}

File: Java/SeleniumTestName.java
package com.selenium;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Main {

    public static void main(String[] args) throws InterruptedException {
	// write your code here
        WebDriver driver = new ChromeDriver();

        driver.get("https://testing.todorvachev.com/selectors/name");

        WebElement element = driver.findElement(By.name("myName"));

        if (element.isDisplayed())
        {
            System.out.printf("Very good! The element appears on the webpage.");
        } else
        {
            System.out.printf("Not good... The element doesn't appear on the webpage");
        }

        Thread.sleep(6000);

        driver.quit();
    }
}
